extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/pages/contract_agreements/services/TransferRequestBuilder.java

```diff
@@ -38,14 +39,15 @@ public class TransferRequestBuilder {
     private final ContractNegotiationUtils contractNegotiationUtils;
     private final EdcPropertyUtils edcPropertyUtils;
     private final TypeTransformerRegistry typeTransformerRegistry;
+    private final ParameterizationCompatibilityUtils parameterizationCompatibilityUtils;
 
     public TransferRequest buildCustomTransferRequest(
             InitiateTransferRequest request
     ) {
         var contractId = request.getContractAgreementId();
         var agreement = contractAgreementUtils.findByIdOrThrow(contractId);
         var negotiation = contractNegotiationUtils.findByContractAgreementIdOrThrow(contractId);
-        var address = edcPropertyUtils.buildDataAddress(request.getDataSinkProperties());
+        var address = parameterizationCompatibilityUtils.enrich(edcPropertyUtils.buildDataAddress(request.getDataSinkProperties()), request.getTransferProcessProperties());
```

Comment:
Also Test with the Custom Json Request

Reply:
Added Units tests for custom Json and EDC management API in DataSourceParameterizationTest